app/scripts/auth/views/login.html

```diff
@@ -1 +1,15 @@
-Login page
+<form novalidate class="login-form">
+
+  <div class="input-group">
+    <input type="text">
+    <label>Username</label>
+  </div>
+
+  <div class="input-group">
+    <input type="password">
+    <label>Password</label>
+  </div>
+
+  <input type="submit" value="Login">
```

Comment:
`button[type=submit]` would be better here, because it behaves more consistent with `a.btn` than `input[type=submit]` does (`input`s can differ visually a bit, which is an annoyance) while keeping the same functionality as `input[type=submit]`. 

As this is intended to look like other button, I'd stick a `.btn` (or whatever we put as a class to button) class to it, to avoid repeating.

    <a class="btn">Some button</a>
will 

    <button type="submit" class="btn">Some button</button>
should look the same in that case